content/en/docs/How Tos/create_upload.md

```diff
@@ -76,3 +76,21 @@ Through the channel, the upload receives from the output paths to local files th
 The only Upload component implemented at the moment, S3, removes those files once uploaded, but there isn't a
 golden rule for what to do with them. This is up to the upload component and should be chosen
 wisely and documented extensively.
+
+## Write tests
+
+Testing an upload component can seem sometimes a bit useless as it probably uses external resources
+and libraries for most of its job.
+
+We thereby provide some suggestions to test those components:
+
+* do not test external libraries when possible, they should be already tested in their packages
```

Comment:
true for unit-tests, but writing an end-to-end test is always a nice thing though